css-two-column-layout/index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Two Column Layout</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+  <div class="container">
+    <div class="row header">
+      <div class="column-full">
+        <h1>SCALE, SPEED, AGILITY</h1>
+        <p>We've dedicated thousands of man hours to making healthcare institutions in the Helium network run 10x more
+          efficiently and deliver and unparalleled quality of care.</p>
+      </div>
+    </div>
+    <div class="row">
+      <div class="column-one-half">
+        <div class="row">
+          <img src="images/circumstance.svg" alt="circumstance" class="image">
+        </div>
+        <div class="row">
+          <h4>INTUITIVE, SMART SOFTWARE</h4>
+          <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Earum ad distinctio culpa saepe doloribus libero
+            quos
+            corrupti nulla iste similique!</p>
+        </div>
+        <div class="row">
+          <img src="images/data.svg" alt="data" class="image">
+        </div>
+        <div class="row">
+          <h4>INTUITIVE, SMART SOFTWARE</h4>
+          <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Earum ad distinctio culpa saepe doloribus libero
+            quos
+            corrupti nulla iste similique!</p>
+        </div>
```

Comment:
Now you have multiple `row` elements, but no `column` elements 🤔 

For this layout, you'll need a `row` inside each `column-half` and two `column` class elements in that `row`, one for the image, and one for the text